### ISSUE TITLE
Fix Dialog Markup

### ISSUE THREAD

**Issue by alizedebray** [swisspost/design-system#6180]
The dialog is missing a `<p>` tag around its text content, required for accessibility.
Also, the action buttons are not correctly indented, they should be on two different lines.
